.gitattributes

```diff
@@ -1,2 +1,9 @@
 # Preserve gradlew's line ending
-gradlew eol=lf
+.gitattributes text eol=crlf
+
+gradlew binary
+
+*.java text eol=crlf
+*.gradle text eol=crlf
+
+* text=auto
```

Comment:
What does this actually change? Why'd you set it?

Reply:
On linux git wants to replace the crlf with lf on all files so I had to set it, this also prevents git from sending .java/.gradle files with other line endings. the `* test=auto` is the default for git to choose the best line ending it sees fit if no other rule overweights it